src/OrchardCore.Modules/OrchardCore.Autoroute/Handlers/AutoroutePartHandler.cs

```diff
@@ -126,6 +131,12 @@ public override async Task UpdatedAsync(UpdateContentContext context, AutorouteP
                     ContentItem = part.ContentItem
                 };
 
+                var cultureAspect = await _contentManager.PopulateAspectAsync(contentItem, new CultureAspect());
```

Comment:
Aha you 're using `PopulateAspectAsync` instead, but I'm not sure wh y `GetContentCultureAsync` did work as expected?!!

I'm expected the `LocalizationPart` the one who responsible for populating the aspect